cmd/monaco/download/download_configs.go

```diff
@@ -233,7 +232,7 @@ func doDownloadConfigs(fs afero.Fs, downloaders downloaders, opts downloadConfig
 func downloadConfigs(downloaders downloaders, opts downloadConfigsOptions) (project.ConfigsPerType, error) {
 	configs := make(project.ConfigsPerType)
 
-	{
+	if shouldDownloadConfigs(opts) {
```

Comment:
double-checked this: this is not actually needed, classic config API download is triggered but the downloader is setup with zero "apis to download" if an "only-settings" is given. 
IMO that setup is pretty confusing and I'd keep the explicit exclusion here where all other types are excluded